casts/row/src/UILink/Logo.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { UIImage } from '@tonlabs/uikit.media';
+import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+import type { LogoProps } from './types';
+import { UIConstant } from '../constants';
+
+export const Logo: React.FC<LogoProps> = ({ logo, loading }: LogoProps) => {
+    if (!logo) {
+        return (
+            <View style={styles.container}>
+                <UISkeleton show={!!loading}>
+                    <View style={styles.image} />
+                </UISkeleton>
```

Comment:
Am I right that the skeleton will look more like a square here than a plain circle (as the logo should look like)?

Reply:
A logo not a circle anymore :)

Reply:
![image](https://user-images.githubusercontent.com/36547799/143458050-2d971655-ee7f-490f-9ed3-d8c500de4048.png)